site/docs/Tutorial-GraphQLClients.md

```diff
@@ -0,0 +1,34 @@
+---
+title: GraphQL Clients
+layout: ../_core/DocsLayout
+category: Tutorial
+permalink: /graphql-js/graphql-clients/
+next: /graphql-js/basic-types/
+---
+
+Since a GraphQL API has more underlying structure than a REST API, there are more powerful clients like [Relay](https://facebook.github.io/relay/) which can automatically handling batching, caching, and other features. But you don't need a complex client to call a GraphQL server. With `express-graphql`, you can just send an HTTP POST request to the endpoint you mounted your GraphQL server on, passing the GraphQL query as the `query` field in a JSON payload.
```

Comment:
I added a curl example. There are several ways to send queries - like you can set content type to application/graphql. I'd rather keep this tutorial simple and just explain one way to do things - I would link to the reference docs for express-graphql, but they don't really exist for this yet. So let's only mention the POST mechanism & put the extensive explanation in reference docs for the 'graphqlHTTP' method later.